test/tests/builtin_type_not_subclassable.py

```diff
@@ -1,3 +1,19 @@
 # should_error
+import re
+
 class foo(slice):
    pass
+
+class C(type(xrange(1))):
```

Comment:
We should try to avoid using `#should_error` now -- it's pretty much only for tests that are supposed to crash.  You could try doing something like.

```python
def is_subclassable(base_cls):
  try:
    class C(base_cls):
      pass
  except TypeError:
    return false
  return true

assert is_subclassable(int)
assert not is_subclassable(slice)
assert not is_subclassable(type(xrange(1)))
```